crates/sp-domains/src/lib.rs

```diff
@@ -398,7 +398,7 @@ sp_api::decl_runtime_apis! {
         ) -> Vec<ExecutionReceipt<NumberFor<Block>, Block::Hash, DomainHash>>;
 
         /// Extract the fraud proofs from the given extrinsics.
-        fn extract_fraud_proofs(extrinsics: Vec<Block::Extrinsic>) -> Vec<FraudProof>;
+        fn extract_fraud_proofs(extrinsics: Vec<Block::Extrinsic>, domain_id: DomainId,) -> Vec<FraudProof>;
```

Comment:
This change is incompatible with current gemini-3a, refer to this example #581 to see how we handle the runtime api change